### ISSUE TITLE
Add `isHidden` alias for `isNotVisible`

### ISSUE THREAD

**Issue by mum-never-proud** [simplabs/qunit-dom#690]
Can we have a straight forward alias named `isHidden` for `isNotVisible`

**Comment by Turbo87** [comment]
yes, we could. problem is that `isVisible()` never really worked properly for all cases and we're considering dropping it completely 😅
